app/vmselect/netstorage/netstorage.go

```diff
@@ -1273,7 +1287,7 @@ func ProcessSearchQuery(qt *querytracer.Tracer, sq *storage.SearchQuery, deadlin
 		putStorageSearch(sr)
 		return nil, fmt.Errorf("cannot finalize temporary file: %w", err)
 	}
-	qt.Printf("fetch unique series=%d, blocks=%d, samples=%d, bytes=%d", len(m), blocksRead, samples, tbf.Len())
```

Comment:
I think it is still useful to keep this kind of reporting here, as it reflects the work to be done